packages/salesforcedx-vscode-core/src/salesforceProject/salesforceProjectConfig.ts

```diff
@@ -61,7 +61,7 @@ export default class SalesforceProjectConfig {
     let errorMessage = error.message;
     if (error.name === 'JsonParseError') {
       errorMessage = nls.localize(
-        'error_parsing_sfdx_project_file',
+        'error_parsing_salesforce_project_file',
```

Comment:
Hmm... the file name is still kept as sfdx-project.json though.  Is this possibly one of the places where changing the variable name would actually make the code less clear?

Reply:
not to me, but I think I'll revert the change just in case